tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -743,6 +745,25 @@ std::unique_ptr<ROOT::Experimental::RNTupleModel> ROOT::Experimental::RNTupleDes
 ////////////////////////////////////////////////////////////////////////////////
 
 
+ROOT::Experimental::RResult<void>
+ROOT::Experimental::RNTupleDescriptorBuilder::EnsureValidDescriptor() const {
+   // Reuse field name validity check
+   Detail::RFieldBase::EnsureValidFieldName(fDescriptor.GetName()).ThrowOnError();
```

Comment:
I think we don't want to throw here but we want to forward the `RResult` if it is an error

Reply:
Should probably also use the facility of #6039

Reply:
Thank you, done in 496701847caa0ea10c96704460d7158b47eb7e7e